website_sale_product_pack/controllers/main.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import http
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class WebsiteSaleProductPack(WebsiteSale):
+    @http.route()
+    def product(self, product, category="", search="", **kwargs):
+        res = super().product(product, category=category, search=search, **kwargs)
+        http.request.env.context = dict(http.request.env.context, whole_pack_price=True)
```

Comment:
Please add docstring about this. I think there's a way of not doing Python hack (maybe through `qcontext`?)

Reply:
It didn't work as expected anyway. I'm removing it